### PR TITLE
Force update project options when assembly references change

### LINKED ISSUE #2020: Editor shows errors when using Framework Reference
This is with RC2.

1. Create a new Console App project.
2. Right-click References node and add a reference to System.Net.Http.
3. Type `open System.Net.Http` in the editor.

**Expected:** No red squiggles.
**Actual:** Red squiggles.

Note that the following code compiles and runs (after adding a reference to System.Net.Http):

```fsharp
// Learn more about F# at http://fsharp.org
// See the 'F# Tutorial' project for more help.

open System.Net.Http

let getHtmlSync (url: string) =
    let client = new HttpClient()
    client.GetStringAsync(url).Result

[<EntryPoint>]
let main argv = 
    printfn "%s" (getHtmlSync "https://google.com")

    printfn "Yay!  It worked!"
    0 // return an integer exit code
```

framework-reference-fail

It also works in FSI if I send the reference down to the FSI process, so this seems to be strictly an editor problem.

### PR DESCRIPTION
**Pull request by dungpa**
Fix #2020.

### CODE REVIEW

**Comment by KevinRansom** [comment]
Oky I think I fished the assembly reference thing again ... packages.config got overwritten.
@dotnet-bot test this please

File: vsintegration/src/FSharp.Editor/LanguageService.fs [hunk 1]
```diff
@@ -207,13 +207,15 @@ type internal FSharpLanguageService(package : FSharpPackage) as this =
     let projectInfoManager = this.Package.ComponentModel.DefaultExportProvider.GetExport<ProjectInfoManager>().Value
 
     /// Sync the information for the project 
-    member this.SyncProject(project: AbstractProject, projectContext: IWorkspaceProjectContext, site: IProjectSite) =
+    member this.SyncProject(project: AbstractProject, projectContext: IWorkspaceProjectContext, site: IProjectSite, forceUpdate) =
```
Comment (KevinRansom) [inline vsintegration/src/FSharp.Editor/LanguageService.fs]: Could we make forceUpdate and enum please, true isn't really readable.

Comment (dungpa) [inline vsintegration/src/FSharp.Editor/LanguageService.fs]: `SyncProject` is a class member with two usages at the same file. At the both call sites, I have used named parameters for `forceUpdate`. I don't think changing bool to two-case enums would improve readability here.

**Comment by cartermp** [comment]
Awesome.  Thanks, @dungpa!

**Pull request merged**